### DISCUSSION TITLE
2 ideas that I think would be good to implement

### DISCUSSION THREAD

**Discussion by valimikayilov** [10x-development/lockmeout.online#6] [Ideas]
1st idea:
Instead of generating password we could upload our own password, this would help a lot. As I see many people online looking for an option that would hide their password from them for some set time. I think it is very inconvenient to have to change the password of your account every time and might lead to a ban on the account.

2nd idea:
Once the time is due you get the image and there is nothing else. But, it would be really nice if we could extend the timer further, instead of just getting the password.

there are could be 2 scenarios, extending the time once the timer is finished and extending while the timer is still going on, in my opinion both would be very useful

**Comment by Dynalon** [comment]
Actually, in an earlier development version I had a feature to generate random passwords and put in your own ones - I removed it because the site/UX flow became too complex. This flow could be solved by bringing back a "free text" field - which is actually still in the code, because I never removed it and you can actually see the form here: https://lockmeout.online/account/new/text - There is just no way to navigate there, but with the deep-link you can see it. It won't allow you to submit, though.

I haven't thought of the issue with the changing passwords, and I believe most of the code is still intact for that feature, I'll see in the next couple of days if I can simply integrate it again.

**Reply by valimikayilov** [reply]
In my opinion, when it comes to **account** lock specifically text is very useful. There are many cases, but one of them is, for example, when I travel and try to change my passwords too frequently my social media account or any other account usually gets locked or banned, because social media or any other account management classifies my account as hacked or similar. if I had a consistent password this occurrence would be avoided, therefore, I think it would be very useful to have this text option.

Also, I have a very long password for an account that I can never remember, instead of changing password consistently I could just store it in a lock.

**Comment by Dynalon** [comment]
Text locks are now implemented as a fourth option in the lock selection screen - let me know if that works for you.

**Reply by valimikayilov** [reply]
for me it is perfect now

**Reply by Dynalon** [reply]
deleted you issue report comment to keep this organized (and fixed it, thanks for the report!). Issues are probably be better reported in the issues tab instead of the discussions  🙏

**Comment by Dynalon** [comment]
I'm closing this, as your first comment was implemented, and the second suggested is now tracked in #9
